packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -64,7 +215,7 @@ class ContinuousRectangleBorder extends ShapeBorder {
     if (a is ContinuousRectangleBorder) {
       return ContinuousRectangleBorder(
         side: BorderSide.lerp(a.side, side, t),
-        borderRadius: BorderRadiusGeometry.lerp(a.borderRadius, borderRadius, t),
```

Comment:
what's the difference here?

Reply:
It's lerping a double instead of the BorderRadiusGeometry struct (which has 8 radii values).